src/Middleware/ValidateSupportedContentTypes.php

```diff
@@ -77,7 +64,6 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
 
     /**
      * @param string|null $contentType
-     * @param ServerRequestInterface $request
      */
     private function matchContentTypeFromRoute($contentType, ServerRequestInterface $request)
```

Comment:
Should we not change this to `?string $contentType` since it already needs to be a string or null?

Reply:
@nesl247 https://github.com/LinioIT/common-expressive/pull/17/commits/7814554de4a9fb8ac83b529b5fe23c8cfe902b2d